frontend/src/morpheus/store/reducers.js

```diff
@@ -56,6 +56,8 @@ export const initialState = {
   }
 };
 
+const normalize = str => str.normalize("NFD").replace(/[\u0300-\u036f]/g, "").toLowerCase();
```

Comment:
Nice job here! :clap: :clap: :clap:

Reply:
i'll make the requested changes and push in a sec